### PR TITLE
feat(ci): move to mc-modpack-kit

### PR DESCRIPTION
**Pull request by sofiedotcafe**

### CODE REVIEW

**Comment by sofiedotcafe** [comment]
@Aninuscsalas You have to set the Modrinth ID and TOKEN in the settings
Also allow "Allow PR's to be managed with actions"

**Comment by sofiedotcafe** [comment]
@Aninuscsalas Can't maybe yet merge cause https://github.com/jh-devv/mc-modpack-kit/issues/49

**Comment by sofiedotcafe** [comment]
@Aninuscsalas This should be pretty much complete (still have to add some stuff) but I want to ask something:

Does the version of the pack need to be different for the Forge and Quilt version?
Because for that, we would need https://github.com/jh-devv/mc-modpack-kit/issues/49

**Comment by aninusmuffin** [comment]
> Does the version of the pack need to be different for the Forge and Quilt version? Because for that, we would need [jh-devv/mc-modpack-kit#49](https://github.com/jh-devv/mc-modpack-kit/issues/49)

I'm still not sure what to do. I think its better to wait because in my test repos modrinth publishing/builing often fails, or skips everything. Im going to some testing with other packs before this one gets merged

**Comment by sofiedotcafe** [comment]
Hmm, weird? Have you read the README?

**Comment by sofiedotcafe** [comment]
> > Does the version of the pack need to be different for the Forge and Quilt version? Because for that, we would need [jh-devv/mc-modpack-kit#49](https://github.com/jh-devv/mc-modpack-kit/issues/49)
> 
> I'm still not sure what to do. I think its better to wait because in my test repos modrinth publishing/builing often fails, or skips everything. Im going to some testing with other packs before this one gets merged

Could you share a test repo of yours?

**Comment by sofiedotcafe** [comment]
@Aninuscsalas First, sorry for so late response!
I got it working? Perhaps you had older releases but it just works fine? Can yoy recheck this?

**Comment by sofiedotcafe** [comment]
This theorically should be now working (the release please things just means you can bump one part of the monorepo)
Shoudl work!

**Comment by aninusmuffin** [comment]
> This theorically should be now working (the release please things just means you can bump one part of the monorepo) Shoudl work!

Should we try merging it? I have added the secrets to the repo. I guess the versioning can restart from 0.1.0... Im still not sure how should i style commit names, chore(quilt):,  and feat(forge): can work fine but im not sure that listing all the updated mods separately or just saying "Updated mods" is a better idea.

**Comment by sofiedotcafe** [comment]
> Should we try merging it? I have added the secrets to the repo. I guess the versioning can restart from 0.1.0... Im still not sure how should i style commit names, chore(quilt):, and feat(forge): can work fine but im not sure that listing all the updated mods separately or just saying "Updated mods" is a better idea.

@Aninuscsalas Yea, but you could just try to run the version workflow right away and set it to just some value (e.g. fabric, it does not support currentry different versions for different components) and for the version set `0.1.0` if you want that!

And for the mod commits, I would use it like:
For updates:
`chore(fabric): Update mods`
For bigger mod changes (e.g. adding or removing)
`feat(fabric): Added mods` or `feat(fabric): Added Fabric API`
Its pretty much how'd you like it!

Hope this helps
*jh-devv*

File: quilt/mods/vow.pw.toml
Comment (aninusmuffin) [inline quilt/mods/vow.pw.toml:1]: This was a bad idea, this needs an access code

Comment (sofiedotcafe) [reply quilt/mods/vow.pw.toml:1]: Yea.. Modrinth does not allow download from "custom" urls

Comment (aninusmuffin) [reply quilt/mods/vow.pw.toml:1]: nono, its on modrinth now https://modrinth.com/mod/vow, it just needs an acces key to use it

Comment (sofiedotcafe) [reply quilt/mods/vow.pw.toml:1]: Oh, cool :3

Comment (sofiedotcafe) [reply quilt/mods/vow.pw.toml:1]: > nono, its on modrinth now https://modrinth.com/mod/vow, it just needs an acces key to use it

Is it still up?

Comment (sofiedotcafe) [reply quilt/mods/vow.pw.toml:1]: Screenshot_20230926_143834
Screenshot_20230926_144025
I asked Rafii about this ^ :3

**Comment by sofiedotcafe** [comment]
@Aninuscsalas Now my project uses commits like
- `feat(modpack/<mod>): <description>` for adding new features, configs or content.
- `chore(modpack/<mod>): <description>` for routine maintenance tasks or cleanups.
- `update(modpack/<mod>): update <mod>` for updating mods to newer versions.
- `mod(modpack/<mod>): add <mod>` for introducing new mods to the modpack.

**Comment by aninusmuffin** [comment]
I'm gonna probably try to merge this this in the next few days 🙃

**Review by aninusmuffin — APPROVED** [review]

**Comment by sofiedotcafe** [comment]
@Aninuscsalas now just run the set version workflow ;3

**Comment by aninusmuffin** [comment]
> @Aninuscsalas now just run the set version workflow ;3

can i do that before merging? (im new to github)

**Comment by sofiedotcafe** [comment]
> > @Aninuscsalas now just run the set version workflow ;3
> 
> can i do that before merging? (im new to github)

You can do it right after or I can do it :3

**Comment by sofiedotcafe** [comment]
https://github.com/orgs/community/discussions/25488 @Aninuscsalas

**Pull request merged**